virttest/qemu_vm.py

```diff
@@ -1890,6 +1890,11 @@ def sort_key(dev):
                                       dynamic=True)
                 devices.insert(pvpanic_dev)
 
+        # Add vmcoreinfo device
+        if params.get("vmcoreinfo") == "yes":
+            vmcoreinfo_dev = qdevices.QDevice("vmcoreinfo")
```

Comment:
Before adding this device, it's better to check if qemu also supports this device as well.
`devices.has_device("vmcoreinfo")`

Reply:
Right, that's more solid in case someone wrongly configure it for unsupported platform. Thanks @PaulYuuu

Reply:
@PaulYuuu  Updated, thanks~

Reply:
And check the function with case, works normally:
PASS 1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.fwcfgdmp.q35